### DISCUSSION TITLE
How to use null character as an UDP input separator?

### DISCUSSION THREAD

**Discussion by ghost** [fluent/fluent-bit#8838] [Q&A]
Hi all,
is there possibility to set null character `\0` as a separator for UDP input?

**Comment by patrick-stephens** [comment]
Did you try specifying it as the `separator` per the docs https://docs.fluentbit.io/manual/pipeline/inputs/udp? Did that not work?

**Reply by ghost** [reply]
I tried that but it didn't work.
